### PR TITLE
Updates from bedrock

### PR DESCRIPTION
**Pull request by MozmarRobot**
From file changes in https://github.com/mozilla/bedrock/commit/916fc374f5d3f8110b1bc254f985aa34c942e1dc

### CODE REVIEW

File: en/firefox/browsers/mobile/focus.ftl [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+mobile-focus-easily-erase-your = Easily erase your history, passwords and cookies, so unwanted ads don’t follow you around online. Just tap the erase button on the search field and it’s gone.
+
+mobile-focus-take-private-mode = Take private mode to the next level
+mobile-focus-private-browsing-on = “Private browsing” on most browsers isn’t really all that private. { -brand-name-firefox-focus } is next-level, always on privacy and it’s backed by { -brand-name-mozilla }, the non-profit that fights for your rights on the web.
+
+mobile-focus-tracking-protection = Tracking protection
+mobile-focus-firefox-focus-blocks = { -brand-name-firefox-focus } blocks a wide range of common trackers by default including social trackers and those sticky ones that come from things like { -brand-name-facebook } ads.
+
+mobile-focus-ad-blocking = Ad blocking
+mobile-focus-focus-blocks-the = { -brand-name-focus } blocks the ads as you’d normally see on web pages — no extra add-ons or extensions necessary. With fewer ads you get better page load speed, which means you get to the stuff you want a lot faster.
```
Comment (peiying2) [inline en/firefox/browsers/mobile/focus.ftl:33]: Add a comma after `With fewer ads`,

File: en/firefox/browsers/mobile/index.ftl [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+browsers-mobile-at-home-or-on-the-move = At home or on the move, you deserve privacy on every device. Enhanced Tracking Protection is automatic in every { -brand-name-firefox } mobile browser.
+
+# The strong tag wraps a word that gets highlighted with a colorful underline for emphasis. The underline breaks if it is on two words; please omit the strong tags if they need to be around multiple words in your language.
+browsers-mobile-get-the-mobile-browser-privacy = Get the mobile browser that puts your <strong>privacy</strong> first
+browsers-mobile-firefox-for-android = { -brand-name-firefox } for { -brand-name-android }
+browsers-mobile-infinitely-customizable-private = Infinitely customizable, private and secure, { -brand-name-firefox } for { -brand-name-android } is a lightning-fast browser that will never sell you out.
+browsers-mobile-firefox-for-ios = { -brand-name-firefox } for { -brand-name-ios }
+
+# "Enhanced Tracking Protection" is a feature name and should be capitalized.
```
Comment (peiying2) [inline en/firefox/browsers/mobile/index.ftl:20]: Many languages don't have lower or upper cases. Shall we change it to `"Enhanced Tracking Protection" is a feature name and should match what's used in the product.`

File: en/firefox/browsers/mobile/compare.ftl [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+# The strong tag wraps a word that gets highlighted with a colorful underline for emphasis. If the translation requires multiple words, please choose a word to emphasize and wrap that word in the strong tag.
+mobile-compare-syncing-strong = <strong>Syncing</strong>
+mobile-compare-syncing = Syncing
+mobile-compare-does-your-browser = Does your browser play well with other devices?
+mobile-compare-the-mobile-version = The mobile version of { -brand-name-safari }, which is pre-installed on { -brand-name-apple }’s mobile devices, is only compatible with other { -brand-name-apple } devices like { -brand-name-ipad } and laptops. { -brand-name-firefox }, { -brand-name-chrome }, and { -brand-name-opera } work across all platforms including { -brand-name-ios }, { -brand-name-android }, { -brand-name-windows }, { -brand-name-mac-short }, and { -brand-name-linux }, with the exception of the { -brand-name-edge } browser which isn’t compatible with { -brand-name-linux }-based devices.
```
Comment (peiying2) [inline en/firefox/browsers/mobile/compare.ftl:31]: Should we mention iPhone here?

File: en/firefox/browsers/mobile/compare.ftl [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+mobile-compare-mobile-os-availability = Mobile OS availability
+mobile-compare-syncs-with-desktopmobile = { -brand-name-sync }s with desktop/mobile app
+mobile-compare-password-management = Password management
+mobile-compare-primary-password = Primary password
+
+mobile-compare-get-firefox-for-mobile = Get { -brand-name-firefox } for mobile
+mobile-compare-send-a-download-link-to-your = Send a download link to your phone.
+mobile-compare-scan-the-qr-code-to-get-started = Scan the QR code to get started
+mobile-compare-scan-alt-text = Scan the QR code to get { -brand-name-firefox } on your mobile device
+
```
Comment (peiying2) [inline en/firefox/browsers/mobile/compare.ftl:64]: At the end of this page: https://www-dev.allizom.org/en-US/firefox/browsers/mobile/compare/ - there is a blurb on the versions being compared and how often this page might be updated. Are we intentionally omitting it for l10n?

**Review by peiying2 — CHANGES_REQUESTED** [review]
A few change request here and there, overall it looks good.

File: en/firefox/browsers/mobile/compare.ftl (hunk 3 above)
Comment (craigcook) [reply en/firefox/browsers/mobile/compare.ftl:31]: Maybe iPhone is implied, but I do think we should specify "Mac laptops". I'll add that.

File: en/firefox/browsers/mobile/compare.ftl (hunk 4 above)
Comment (craigcook) [reply en/firefox/browsers/mobile/compare.ftl:64]: The blurb uses existing strings from firefox/browsers/compare/shared.ftl

**Review by peiying2 — APPROVED** [review]
Thank you for the updates. Looks good.

**Pull request merged**